{{ cookiecutter.__project_name_kebab_case }}/README.md

```diff
@@ -1,19 +1,47 @@
-[![Open in Dev Containers](https://img.shields.io/static/v1?label=Dev%20Containers&message=Open&color=blue&logo=visualstudiocode)](https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url={{ cookiecutter.package_url.replace("https://", "git@").replace(".com/", ".com:") if cookiecutter.private_package_repository_url else cookiecutter.package_url }})
+[![Open in Dev Containers](https://img.shields.io/static/v1?label=Dev%20Containers&message=Open&color=blue&logo=visualstudiocode)](https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url={{ cookiecutter.project_url.replace("https://", "git@").replace(".com/", ".com:") if cookiecutter.private_package_repository_url else cookiecutter.project_url }})
 
-# {{ cookiecutter.package_name }}
+# {{ cookiecutter.project_name }}
 
-{{ cookiecutter.package_description }}
+{{ cookiecutter.project_description }}
+{%- if cookiecutter.project_type == "package" or cookiecutter.with_typer_cli|int %}
 
-## Using
+## Installing
+
+To install this package, run:
 
-{% if cookiecutter.with_fastapi_api|int or cookiecutter.with_typer_cli|int or cookiecutter.with_streamlit_app|int %}_Python package_: t{% else %}T{% endif %}o add and install this package as a dependency of your project, run `poetry add {{ cookiecutter.__package_name_kebab_case }}`.
+```sh
+pip install {{ cookiecutter.__project_name_kebab_case }}`
+```
+{%- endif %}
+
+## Using
 {%- if cookiecutter.with_typer_cli|int %}
 
-_Python CLI_: to view this app's CLI commands once it's installed, run `{{ cookiecutter.__package_name_kebab_case }} --help`.
+To view the CLI help information, run:
+
+```sh
+{{ cookiecutter.__project_name_kebab_case }} --help
+```
+{%- elif cookiecutter.project_type == "app" -%}
+To serve this app, run:
+
+```sh
+docker compose up app
+```
+{%- if cookiecutter.with_fastapi_api|int %}
+
+and open [localhost:8000](http://localhost:8000) in your browser.
 {%- endif %}
-{%- if cookiecutter.with_fastapi_api|int or cookiecutter.with_streamlit_app|int %}
 
-_Python application_: to serve this {% if cookiecutter.with_fastapi_api|int %}REST API{% else %}Streamlit app{% endif %}, run `docker compose up app` and open [localhost:8000](http://localhost:8000) in your browser. Within the Dev Container, this is equivalent to running {% if cookiecutter.with_fastapi_api|int %}`poe api`{% else %}`poe app`{% endif %}.
+Within the Dev Container, this is equivalent to running `poe {% if cookiecutter.with_fastapi_api|int %}api{% else %}app{% endif %}`.
```

Comment:
```suggestion
Within the Dev Container, this is equivalent to running 

```sh
poe {% if cookiecutter.with_fastapi_api|int %}api{% else %}app{% endif %}`.
```
```

Reply:
I think the suggestion is missing triple backticks at the end, but I got the point. Will improve!